samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -1445,7 +1362,6 @@ static void light_ctl_temp_get(struct bt_mesh_model *model,
 	struct light_ctl_state *state = model->user_data;
 
 	bt_mesh_model_msg_init(msg, BT_MESH_MODEL_OP_2(0x82, 0x66));
-
```

Comment:
This looks like a coding style fix. Those should be in a separate commit. Same for any other style fixes in this PR.